### PR TITLE
fix/handle error when metamask not installed

### LINKED ISSUE #77: bug: uncaught error when metamask is not installed
When the browser don't have metamask connected there is an error but not enough explicit for the user

### Acceptance criteria

- [ ] A Pop Up to request the user to install metamask (or a wallet) with a link to install it.

### PR DESCRIPTION
**Pull request by dasarathg68**
# Description

Fixes #77 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

### CODE REVIEW

**Comment by hermannleboss** [comment]
@dasarathg68 i add accepatance criteria on this

File: app/src/adapters/web3LibraryAdapter.ts
Comment (hermannleboss) [inline app/src/adapters/web3LibraryAdapter.ts:1]: Every time you call the function connectWallet you can have an error throw new Error('Wallet is not connected')
Need more test about how the error is handle and what are the error we can have

**Review by hermannleboss — CHANGES_REQUESTED** [review]

**Comment by farrellh1** [comment]
u can use new library in #75

**Review by hermannleboss — APPROVED** [review]
LGTM

**Pull request merged**